### DISCUSSION TITLE
Docker auto update using cron

### DISCUSSION THREAD

**Discussion by rquara** [AzuraCast/AzuraCast#4215] [Q&A]
I want to automatically check every night if there is a new version of the azuracast docker and automatically update it.

I don't know much about cron but I guess it should be something like this
0 4 * * *./var/azuracast/docker.sh update-self && yes "" | ./var/azuracast/docker.sh update

Unfortunatly it isn't working and I'm not sure about what is wrong about it.

I was also thinking about an automatic backup of the docker before the cron execution but I really don't know how to do it.

Any suggestion? Thanks in advance

**Comment by ItsMitchh** [comment]
Without seeing your full code, I can offer no suggestions.

**Reply by rquara** [reply]
what full code are you referring to? I'm just using the standard docker upgrade process reccomanded here https://docs.azuracast.com/en/getting-started/updates

**Comment by BusterNeece** [comment]
@rquara Crontabs are _really_ picky about using the full path to things, and I think you're using a relative path where you don't need to be, maybe it should look like:

```
0 4 * * * bash /var/azuracast/docker.sh update-self && yes "" | bash /var/azuracast/docker.sh update
```

**Reply by rquara** [reply]
thanks for the suggestion. I tried to add it to the crontab but it didn't work. No update was performed.
